static/reference/next/config/README.md

```diff
@@ -32,3 +32,72 @@ You can set specific configurations through Java properties.
 | **Property**       | **Description**                                              |
 | ------------------ | ------------------------------------------------------------ |
 | pulsar.enableUring | Use `io_uring` instead of `epoll` as network IO mode. `-Dpulsar.enableUring=1` means this feature is enabled. |
+
+## Broker load balancing configurations
```

Comment:
For reviewers:

The detailed descriptions of broker load balancing configurations are shown at https://pulsar.apache.org/reference/#/next/config/reference-configuration-broker. This page is generated from code files automatically. All configurations including those related to broker load balancing are listed on that page without appropriate sorting. 

To give users a clear and quick understanding of broker load balancing configurations, I've added this brief summary.